assets/static/css/style.css

```diff
@@ -0,0 +1,66 @@
+.container {
+  width: 80%;
+  max-width: 75rem;
+  margin: 0 auto;
+}
+
+.title {
+  text-align: center;
+}
+
+.flex-outer {
+  list-style-type: none;
+  padding: 0;
+  max-width: 50rem;
+  margin: 0 auto;
+}
+
+.flex-outer .content-row {
```

Comment:
Do we need to couple content-row and flex-outer? Seems like they dont need to be tied together here.

Reply:
Right, i guess I can remove this prefixing with .flex-outer in a few other places as well, it was an attempt to be more explicit but I'm not well versed on the best practices. I guess its safe to remove considering we arent planning to add more to this form.